libcodechecker/libhandlers/analyze.py

```diff
@@ -137,6 +137,16 @@ def add_arguments_to_parser(parser):
                         help="Annotate the ran analysis with a custom name in "
                              "the created metadata file.")
 
+    parser.add_argument('-f', '--force',
+                        dest="force",
+                        required=False,
+                        action='store_true',
+                        default=False,
+                        help="Delete analysis reports stored in the output "
+                             "directory. (By default, CodeChecker would keep "
+                             "reports and overwrites only those plist files "
```

Comment:
Because the type of files is a command-line argument, I would not use the word `plist` here.

Also, let's move this command between `--type` and `--name`. The order of arguments matter, and this is more logical to follow the "output" arguments.